server/routes/questions/router.test.ts

```diff
@@ -372,6 +394,31 @@ describe(`Submitting questions' responses`, () => {
       },
     ]
 
+    const questionsResponse: Question[] = [
+      {
+        code: '67179',
+        question: 'DESCRIBE HOW THE ITEM WAS FOUND (SELECT ALL THAT APPLY)',
+        additionalInformation: null,
+        responses: [
+          {
+            response: 'BOSS CHAIR',
+            responseDate: new Date(),
```

Comment:
technically `responseDate` is null (and for next response too)

Reply:
True, these answers have `commentRequired: false` (although in practice it doesn't make a difference for the test)

Reply:
it totally doesn't make any practical difference. but realistic test/mock data structures make it more readable and easier to reason about… otherwise i might look in 2 months time and wonder where the date came from

Reply:
Changed test data to have `null` dates when they're not required in this commit: https://github.com/ministryofjustice/hmpps-incident-reporting/pull/301/commits/e2c1965538098f906eb63c448ba85c00e6adda0b